docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md

```diff
@@ -55,49 +55,12 @@ Get transfer/s Participant/s details.
 
 #### Description
 
-Get transfer/s Party/s details.
+Where a participant DFSP requests Party details, this UC is used by the switch to validate the request and provide the requested relevant Party data to the requesting DFSP.
```

Comment:
```suggestion
Where a participant DFSP requests Party information based on a Parties identifier.
```

Reply:
@elnyry-sam-k please review and provide your input here.

Reply:
looks good Michael, thank you..

Reply:
no problem Michael, have a great weekend; it's no problem, can continue till we resolve the open issues :-) ..